frontend/src/components/material/MaterialTable.vue

```diff
@@ -28,13 +28,13 @@
     </template>
 
     <template #[`item.quantity`]="{ item }">
-      <api-text-field
+      <api-number-field
```

Comment:
This seems to not yet work.
[Screencast from 27.04.2024 14:03:30.webm](https://github.com/ecamp/ecamp3/assets/1506818/346f7b4b-72dc-4e18-8552-60b949619ea8)

Reply:
Can you please test it in this deployment? :D You're testing in #5025